crates/rpc-types-trace/src/geth/mod.rs

```diff
@@ -130,6 +132,8 @@ pub enum GethTrace {
     NoopTracer(NoopFrame),
     /// The response for mux tracer
     MuxTracer(MuxFrame),
+    /// The response for ERC-7562 tracer
+    Erc7562Tracer(Erc7562Frame),
```

Comment:
this we can't do just yet because this is a breaking change

Reply:
you can undo the changes for this pr and send another pr that adds these variants